### PR TITLE
Trade chat post timer

### PR DESCRIPTION
**Pull request by Sashank999**
This PR also contains some changes to `addPeopleBoxFilter()`.

### CODE REVIEW

File: torntools/scripts/content/global/ttGlobal.css [hunk 1]
```diff
@@ -1006,3 +1006,22 @@ div[role="main"].tt-modified {
 .tt-blink {
 	background-color: #ff0a0ab3;
 }
+
+.tt-trade-chat-timer {
+	margin-top: 0px;
```
Comment (DeKleineKobini) [inline torntools/scripts/content/global/ttGlobal.css:1011]: 0

File: torntools/scripts/content/global/ttGlobal.css [hunk 2]
```diff
@@ -1006,3 +1006,22 @@ div[role="main"].tt-modified {
...
+    height: 15px;
+    width: max-content;
+    float: left;
+    margin-left: 5px;
+    color: white;
+    border-radius: 5px;
+    padding: 2px 2px 0px 2px;
```
Comment (DeKleineKobini) [inline torntools/scripts/content/global/ttGlobal.css:1018]: 0

File: torntools/scripts/content/global/ttGlobal.js [hunk 3]
```diff
@@ -754,8 +756,8 @@ function nukeReviveScript() {
 
 // Code for adding chat filter for People Box
 function addPeopleBoxFilter() {
-	if (document.find(".chat-box-people_SPbEh").findAll(".tt-chat-filter").length === 0) {
-		let peopleBox = document.find(".chat-box-people_SPbEh");
+	if (doc.findAll("div[class*='chat-box-people_SPbEh'] .tt-chat-filter").length === 0) {
```
Comment (DeKleineKobini) [inline torntools/scripts/content/global/ttGlobal.js:759]: Why would you use `*=` and still use the full class? Either do `.class_xxx` or `[class*='class_']`.

File: torntools/scripts/content/global/ttGlobal.js [hunk 4]
```diff
@@ -754,8 +756,8 @@ function nukeReviveScript() {
...
+		let peopleBox = document.find("div[class*='chat-box-people_SPbEh']");
```
Comment (DeKleineKobini) [inline torntools/scripts/content/global/ttGlobal.js:760]: same as above

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
Changelog

**Comment by Sashank999** [comment]
Resolved them all.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
Still missing changelog.
-- Not tested after changes yet.

**Comment by Sashank999** [comment]
Changelog added.

**Pull request merged**